src/aap_eda/api/views/activation.py

```diff
@@ -13,7 +13,7 @@
 #  limitations under the License.
 import logging
 
-import redis
+# redis import removed - no longer required with dispatcherd migration
```

Comment:
Does it makes sense these kind of code comments? Code comments are usually intended to describe or clarify the code. For historical purposes I think there are better choices like the release notes (or maybe some announce in the docs) or the VCS itself.